resources/phing/AtoumTask.php

```diff
@@ -287,13 +309,37 @@ public function getCodeCoverage()
 		return $this->codeCoverage;
 	}
 
+	public function setTelemetry($telemetry)
```

Comment:
I would have split this into two method: 
* `enableTelemetry()`
* `disableTelemetry()`